ci/jenkins/test.sh

```diff
@@ -682,6 +699,11 @@ function run_e2e {
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+        ./hack/generate-manifest.sh | docker exec -i ${CLUSTER}-control-plane dd of=/root/antrea.yml
+        kubectl apply -f build/yamls/antrea-prometheus.yml
+        kubectl apply --context kind-${CLUSTER} -f $THIS_DIR/../../build/yamls/antrea-prometheus-rbac.yml
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind --kind.kubeconfig "${CLUSTER_KUBECONFIG}" -timeout=100m --prometheus
```

Comment:
The github workflow .github/workflows/kind.yml also runs E2E test in kind via ./ci/kind/test-e2e-kind.sh. Maybe we don't need to run Antrea E2E in kind via this test.sh.